### ISSUE TITLE
The scoreboard is discolored

### ISSUE THREAD

**Issue by tncc2k** [Puharesource/TitleManager#312]
Spigot build 1.10.2
Titlemanager version: 2.3.1

config scoreboard:

```
scoreboard:

  # Toggles this feature
  enabled: true

  # The title displayed at the very top of the scoreboard
  # WARNING! (1.12 or below ONLY) The title must consist of 32 or less characters (this includes color codes)
  title: '${shine:[0;2;0][0;25;0][0;25;0][&4&l;&c&l]aaaaaaaaa}'

  # The lines of the scoreboard (Maximum of 15 lines allowed)
  lines:
  - '&a----------------------'
  - '&a⦿ Money: %vault_eco_balance_fixed%'
  - '&a----------------------'
  - '&aaaaaaaaaaaaaaaabdfdfdfsdff'
```
![titleloi](https://user-images.githubusercontent.com/77291313/105435706-6dac8180-5c90-11eb-9eb6-994a7b240426.jpg)
![config](https://user-images.githubusercontent.com/77291313/105435815-9c2a5c80-5c90-11eb-8581-8e36f27259f0.jpg)

**Comment by ham1255** [comment]
this actual bug in minecraft versions before 1.13 and it was fixed in 1.13+ so only fix here update to newer version

**Comment by Puharesource** [comment]
What @ham1255 said
